mats/misc.ms

```diff
@@ -4952,18 +4952,22 @@
   (begin
     (define (procedure-name f)
       (((inspect/object f) 'code) 'name))
+    (define (ok-name? name expect)
+      (or (equal? name expect)
+	  ;; interpreter currently doesn't keep names
+	  (equal? (current-eval) interpret)))
```

Comment:
Suggestion: use `eq?` here and indent lines 4957-8 to line up with 4956.

Reply:
Done: switched to `eq?` and untabified.